### ISSUE TITLE
Does Not Build

### ISSUE THREAD

**Issue by nkanetka** [ollyau/EFBConnect#1]
@ollyau I cloned the repo and tried to build it with Visual Studio 2015 but I got an error as the BeatlesBlog.SimConnect can not be found. I tried looking for a way to download this but it seems like most download links are dead.

If you could build the project, and make the executable, or provide a way to get the BeatlesBlog.SimConnect module that would be great!

**Comment by ollyau** [comment]
I'll build it locally and upload a release when I get a chance.  In the meantime, I have a link saved from when SkyDrive URLs still redirected to OneDrive; this is what was linked on Tim Gregson's blog for BeatlesBlog.SimConnect (beta 3): https://onedrive.live.com/download?resid=B48B0194C5456F5%21353.

Also, out of curiosity, how did you come across my utility?  Glad to see someone (hopefully) getting some use out of it. :)

**Comment by nkanetka** [comment]
@ollyau Thanks! I got it working using your link. If I get time, I'll fork the repo and make a PR adding the link to the Readme or add them right into the repo (will save someone a ton of time searching if they use this)?

As for how I got here, I was looking for a way to test an application that I made and looking up 'Microsoft Flight Simulator position' in Github resulted in your repo as the top result 🥇.

Thanks again!
